cellprofiler/modules/correctilluminationcalculate.py

```diff
@@ -206,209 +226,222 @@ def create_settings(self):
+   image or 30 pixels, whichever is smaller.
+-  *%(FI_OBJECT_SIZE)s:* The size is obtained relative to the width of
+   artifacts to be smoothed.
+-  *%(FI_MANUALLY)s:* Use a manually entered value.
+""" % globals())
 
         self.object_width = cps.Integer(
-                "Approximate object size", 10, doc='''
-            <i>(Used only if %(FI_AUTOMATIC)s is selected for smoothing filter size calculation)</i><br>
-            Enter the approximate width of the artifacts to be smoothed, in pixels.''' % globals())
+            "Approximate object size", 10, doc="""\
+*(Used only if %(FI_AUTOMATIC)s is selected for smoothing filter size calculation)*
+Enter the approximate width of the artifacts to be smoothed, in pixels.
+""" % globals())
 
         self.size_of_smoothing_filter = cps.Integer(
-                "Smoothing filter size", 10, doc='''
-            <i>(Used only if %(FI_MANUALLY)s is selected for smoothing filter size calculation)</i><br>
-            Enter the size of the desired smoothing filter, in pixels.''' % globals())
+            "Smoothing filter size", 10, doc="""\
+*(Used only if %(FI_MANUALLY)s is selected for smoothing filter size calculation)*
+Enter the size of the desired smoothing filter, in pixels.
+""" % globals())
 
         self.save_average_image = cps.Binary(
-                "Retain the averaged image?", False, doc='''
-            The averaged image is the illumination function
-            prior to dilation or smoothing. It is an image produced during the calculations, not typically
-            needed for downstream modules. It can be helpful to retain it in case you wish to try several
-            different smoothing methods without taking the time to recalculate the averaged image each time.
-            <p>Select <i>%(YES)s</i> to retain this averaged image. Use the <b>SaveImages</b> module to save
-            it to your hard drive.</p>''' % globals())
+            "Retain the averaged image?", False, doc="""\
+The averaged image is the illumination function prior to dilation or
+smoothing. It is an image produced during the calculations, not
+typically needed for downstream modules. It can be helpful to retain it
+in case you wish to try several different smoothing methods without
+taking the time to recalculate the averaged image each time.
+            
+Select *%(YES)s* to retain this averaged image. Use the **SaveImages**
+module to save it to your hard drive.
+""" % globals())
 
         self.average_image_name = cps.ImageNameProvider(
-                "Name the averaged image", "IllumBlueAvg", doc='''
-            <i>(Used only if the averaged image is to be retained for later use in the pipeline)</i><br>
-            Enter a name that will allow the averaged image to be selected later in the pipeline.''')
+            "Name the averaged image", "IllumBlueAvg", doc="""\
+*(Used only if the averaged image is to be retained for later use in the pipeline)*
```

Comment:
Add a newline between `*(Used only if...)*` and the next line. This will insert a line break in the rendered documentation. Otherwise, the text appears on the same line.